includes/options/class-amp-options-menu.php

```diff
@@ -132,7 +132,7 @@ public function add_menu_items() {
 	 * @since 1.0
 	 */
 	public function render_theme_support() {
```

Comment:
@johnwatkins0 FYI: These changes here are incomplete as I expect you'll be revamping everything as part of #4704.